templates/new/batch_runner.py.txt

```diff
@@ -0,0 +1,23 @@
+from surround import Runner
+from stages import {project_name}Data
+
+
+class BatchRunner(Runner):
+    def prepare_runner(self):
+        # Load data from files or directory here
+        self.raw_data = "TODO: Load raw data here"
+        self.data = {project_name}Data()
+        self.assembler.run_on_batch_mode()
+        self.assembler.init_assembler(self.data)
+
+    def prepare_data(self):
+        # Parse raw data into pipeline ready format here
+        self.data.input_data = self.raw_data
+
+    def run(self, is_training=False):
```

Comment:
We probably don't have to do this now but couldn't this method be included in the `Runner` class? Perhaps we need a file system runner..?